### ISSUE TITLE
swaymsg hook incorrect

### ISSUE THREAD

**Issue by sidju** [Kharacternyk/pacwall#46]
The packaged swaymsg hook in incorrectly quoted.
Additionally I couldn't get the $W environment variable to work.

fyi, here is my current hook
```
hook: "swaymsg 'output * bg ~/.cache/pacwall/pacwall.png fit #073642'"
```

**Comment by sidju** [comment]
Note that I use 'fit' instead of 'center' for the positioning and scaling. This makes the whole image show up.

**Comment by ferdinandschober** [comment]
was just about to open an issue:
correct way is
hook: "swaymsg output \* bg $HOME/.cache/pacwall/pacwall.png center '#073642'"

**Comment by sidju** [comment]
And with that tip we can conclude that it should be 
```
hook: "swaymsg output \* bg $W fit #073642"
```
At least that works for me.

**Comment by Kharacternyk** [comment]
You are free to choose your hook, but I suggest to not use `fit` because it upscales/underscales a raster image. A better solution IMHO is to use `center` and change the graphviz attributes related to size or change the DPI that graphviz uses.

I'll fix the example hook ASAP, thanks.

**Comment by Kharacternyk** [comment]
I've hopefully fixed this in 3c24bb0. I'll be grateful if someone tests it.
